src/pages/patientView/copyNumberAlterations/column/CustomDriverColumnFormatter.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { DiscreteCopyNumberData } from 'cbioportal-ts-api-client';
+import _ from 'lodash';
+import { AnnotatedNumericGeneMolecularData } from 'shared/model/AnnotatedNumericGeneMolecularData';
+import { DriverFilterOrder } from 'shared/lib/StoreUtils';
+
+/**
+ * Discrete Copy Number Formatter.
+ */
+export default class CustomDriverColumnFormatter {
+    public static getTextValue(data: DiscreteCopyNumberData[]): string {
+        let textValue: string = '';
+        const dataValue = CustomDriverColumnFormatter.getData(data);
+
+        if (dataValue) {
+            textValue = dataValue.toString();
+        }
+
+        return textValue;
+    }
+
+    public static getData(data: DiscreteCopyNumberData[]) {
+        if (data.length > 0) {
```

Comment:
maybe put a comment explaining this.    also, you can use _.isEmpty()

Reply:
resolved